app/Http/Controllers/Assets/AssetsController.php

```diff
@@ -508,7 +508,7 @@ public function getQrCode($assetId = null) : Response | BinaryFileResponse | str
         if ($settings->qr_code == '1') {
             $asset = Asset::withTrashed()->find($assetId);
             if ($asset) {
-                $size = Helper::barcodeDimensions($settings->barcode_type);
```

Comment:
This line caught my eye since we're passing in a different variable but I _think_ it's ok because `barcodeDimensions` has an `else` that falls back to existing behavior.

Reply:
Yea, so `barcode_type` refers to 2d/QR barcode type. both the new (`2d_barcode_type`) and `barcode_type are an array of the save values, so this really is just a 1:1 swap of info. We just removed the duplicate array/list here.